Exercise_4/docs/part-1.md

```diff
@@ -0,0 +1,330 @@
+
+
+### EHR Technical Architecture
+To better understand the data elements contained in an EHR, it is important to understand the technical architecture of an EHR system. This section describes the major architectural components of an EHR system, and the interactions between these components.
+
+
+![EHR System Components](img/ehr_system.png)
+
+Source [https://confluence.ihtsdotools.org/display/DOCCDS/2.1.+EHR+System+Architecture](https://confluence.ihtsdotools.org/display/DOCCDS/2.1.+EHR+System+Architecture)
+
+## Access and Regulatory Considerations
+
+**Given the highly sensitive nature of data potentially contained within EHR, significant regulatory and programmatic considerations come into play.**
+
+If you're interested in learning more, check out below:
+
+??? abstract "PII and PHI Information and Concerns"
+    * Before potentially accessing EHR data, it is important to note that such information is considered highly sensitive and is subject to considerable safeguards. Specifically, much of the data contained within an EHR carries a legal designation of Personnally Identifiable Information (PII) and Protected Health Information (PHI).
+    * PII is information that can be used to distinguish or trace an individual's identity, either alone or when combined with other personal or identifying information that is linked or linkable to a specific individual.
+        * Source: Glossary of NIH Terms [https://grants.nih.gov/grants/glossary.htm#P](https://grants.nih.gov/grants/glossary.htm#P)
+    * Individually identifiable health information” is information, including demographic data, that relates to:
+        * the individual’s past, present or future physical or mental health or condition, 
+        * the provision of health care to the individual, or 
+        * the past, present, or future payment for the provision of health care to the individual, and that identifies the individual or for which there is a reasonable basis to believe it can be used to identify the individual.
+	    * Individually identifiable health information includes many common identifiers (e.g., name, address, birth date, Social Security Number).
+    * It should be noted that often researchers will work with de-identified EHR data that has purposely been stripped of PII. This may negate specific privacy concerns but there are other, additional privacy protections and considerations for handling even de-identified data.
+    * There could also be issues if Privacy Preserving Record Linkages are being made on the back end to allow for connections between different systems to identify the same individual to create a more robust dataset.
```

Comment:
```suggestion
    * There could also be privacy issues if multiple records of the same individual are linked together to create a more robust dataset.
 ```

I don't like how this sentence is worded. I was going to rewrite it, but I'm not sure what it is trying to say.